pygmt/src/grdfilter.py

```diff
@@ -119,12 +119,11 @@ def grdfilter(grid, **kwargs):
         - None if ``outgrid`` is set (grid output will be stored in file set by
           ``outgrid``)
 
-    Example
-    -------
+    Examples
+    --------
     >>> import os
     >>> import pygmt
-
-    >>> # Apply a filter of 600km (full width) to the @earth_relief_30m file
+    >>> # Apply a filter of 600 km (full width) to the @earth_relief_30m_g file
     >>> # and return a filtered field (saved as netcdf)
```

Comment:
Apply capitalization to NetCDF perhaps?
```suggestion
    >>> # and return a filtered field (saved as NetCDF)
```